templates/post-create.html

```diff
@@ -0,0 +1,25 @@
+        <!-- <"post edit"> -->
+        <form class="form-horizontal">
+            <div class="panel-body text-center mt-5">
+                <img class="rounded-3 mb-1" src="https://img.freepik.com/free-photo/wide-angle-shot-single-tree-growing-clouded-sky-during-sunset-surrounded-by-grass_181624-22807.jpg" alt="Post">
+            </div>
+            <hr class="mb-0">
+        </form>
+        <div id="title" class="container col-md-5 col-lg-4 col-xl-3 col-sm-6 mx-auto mt-5">
+            <!-- Titlename input -->
+            <div class="form-outline mb-3 input-group flex-nowrap">
+                <span class="input-group-text fa-solid fa-file-signature fs-5" id="addon-wrapping"></span>
+                <input type="text" class="form-control" placeholder="Create Title Name" aria-label="Title" aria-describedby="addon-wrapping">
+            </div>
+            <div class="mb-4">
+                <label for="formFile" class="form-label">File input</label>
+                <input class="form-control " type="file" id="formFile">
+            </div>
+            <!-- Save and Cancel buttons -->
+            <div class="container-fluid">
+                <div class="text-center">
+                    <button type="button" class="btn btn-secondary mb-4 col-5">Cancel</button>
+                    <button type="button" class="btn btn-primary mb-4 col-5">Create</button>
+                </div>
+            </div>
+        </div>
```

Comment:
your inputs should be in form tag.

Reply:
it is from line 2 to  7. or do you mean the whole page ?

Reply:
Your input elements are outside of form tag. 
![image](https://user-images.githubusercontent.com/67695829/216511426-163f07f7-eee9-4b0c-9d67-31c29290de9e.png)